src/main/scala/code/api/berlin/group/v1_3/AccountInformationServiceAISApi.scala

```diff
@@ -162,7 +162,7 @@ As a last option, an ASPSP might in addition accept a command with access rights
 }"""),
        List(UserNotLoggedIn, UnknownError),
        Catalogs(notCore, notPSD2, notOBWG),
-       apiTagAccountInformationServiceAIS :: apiTagMockedData :: Nil
+       ApiTag("Account Information Service (AIS)") :: apiTagMockedData :: Nil
```

Comment:
this "net.liftweb.mapper.LongKeyedMapper" get from swagger-codegen, and maybe multiple, so here make the tags dynamic, instead of constants.